tests/suites/test_suite_ssl.data

```diff
@@ -35,34 +35,16 @@ Test mock socket sanity
 ssl_mock_sanity:
```

Comment:
Commit message title "Non-blocking read/write for mbedtls_mock_socket returns..." is too long. Write a more concise title (try to be less than 50 characters or so) and provide more detail in the commit message body.

Reply:
done

Reply:
Please add more detail in the commit message body about why this change is necessary and useful.

Reply:
I added more description to commit message

Reply:
Typo in the commit message, `blockind` should be `blocking`.

Reply:
eh... corrected